modules/virt-aws-bm.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// * virt/install/preparing-cluster-for-virt.adoc
+
+:_content-type: CONCEPT
+[id="virt-aws-bm_{context}"]
+= {VirtProductName} on AWS bare metal
+
+You can run {VirtProductName} on an Amazon Web Services (AWS) bare metal {product-title} cluster. Before you set up your cluster, review the following summary of supported features and limitations:
+
+Installing::
+
+* You can install the cluster by using installer-provisioned infrastructure, ensuring that you specify bare metal node instance types in the `install-config.yaml` file. For example, you can use the `c6g.metal` type value for a machine based on 64-bit ARM architecture.
+** For more information, see the {product-title} documentation about installing on AWS.
+
+Accessing virtual machines (VMs)::
+
+* There is no change to how you access VMs by using the `virtctl` CLI tool or the {product-title} web console.
+* You can expose VMs by using a `NodePort` or `LoadBalancer` service. 
+** The load balancer approach is preferable because {product-title} automatically creates the load balancer in AWS and manages its lifecycle. A security group is also created for the load balancer, and you can use annotations to attach existing security groups. When you remove the service, {product-title} removes the load balancer and its associated resources.
+
+Networking::
+
+* Because connecting a secondary network to a VM is not supported on AWS, you must use the primary network, which you can expose by using services. 
+* If your application requires a flat layer 2 network or control over the IP pool, consider using OVN-Kubernetes secondary overlay networks.
+* You cannot use Single Root I/O Virtualization (SR-IOV) or bridge Container Network Interface (CNI) networks, including virtual LAN (VLAN).
+
+Storage::
+--
+* Amazon Elastic File System (EFS) is only supported in combination with a software-defined storage (SDS) layer.
```

Comment:
I don't understand this statement. Can we explain better with which SDS layer EFS would be supported? @aglitke 

Also, this statement repeats in the Unsupported Storage section. Is it intentional/advisable @ousleyp ?

Reply:
It was indeed intentional because EFS is both supported and unsupported. 😅 My understanding is that EFS is not supported by itself, but EFS is ok if you use Portworx or some other SDS layer as well. If this isn't the case, I will happily edit this. @aglitke

Reply:
I think it would be better to say that native AWS storage options (EBS and EFS) have limitations and are therefore not recommended for use with OCP-V.  Instead, a software-defined storage solution such as AWS FSx or Portworx should be used.

Reply:
@aglitke ok, so the SDS layer is instead of the native storage, not in addition to it? 

My understanding is that FSx isn't supported for 4.14. Are ODF and Portworx the only supported SDS solutions for this in 4.14?

Reply:
Confirmed with Kedar that Portworx is the only tested storage provider for CNV on AWS BM. Removed the reference to ODF. cc @dankenigsberg